app/src/main/java/com/example/tasky/feature_agenda/data/local/AgendaDao.kt

```diff
@@ -0,0 +1,53 @@
+package com.example.tasky.feature_agenda.data.local
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Query
+import androidx.room.Transaction
+import androidx.room.Upsert
+import com.example.tasky.feature_agenda.data.local.entity.AttendeeEntity
+import com.example.tasky.feature_agenda.data.local.entity.EventEntity
+import com.example.tasky.feature_agenda.data.local.entity.ReminderEntity
+import com.example.tasky.feature_agenda.data.local.entity.TaskEntity
+import com.example.tasky.feature_agenda.data.local.relations.EventWithAttendee
+
+@Dao
+interface AgendaDao {
```

Comment:
I'd separate the different agenda items into separate DAOs, so `TaskDao`, `EventDao` and `ReminderDao`. That keeps things much more organized